data.js

```diff
@@ -0,0 +1,133 @@
+/* jshint esversion: 6 */
+
+const restaurants = [{
+  name: 'W Imbiss',
```

Comment:
I think we should placed the name of the restaurant as const as well, so we use const below for the restaurants from the "people". Doing that we get rid of raw-strings. Actually, I see that you are using the name of the restaurant as ID... what if the name of the restaurant changes? ...
I'm saying this from the perspective of from where are you doing the relation between the restaurant and the people.

Reply:
OK will add that